src/amo/components/Addon/styles.scss

```diff
@@ -185,6 +185,10 @@
       // Fixes screenshot overflow in FF 52 ESR.
       max-width: 100%;
       overflow-x: hidden;
+
+      .Card {
```

Comment:
not sure this is safe, I'd go for a more restrictive rule, like `.Card:first-child` or something similar.